### PR TITLE
Issue #1315 by bartvig: Change overridden view to hook_views_default_views_alter

### PR DESCRIPTION
**Pull request by bartvig**
Jeg har tilføjet ændringerne som et views_default_views_alter-hook og har lavet en update-funktion, der disabler det gamle view og enabler viewet fra admin_views.

Jeg har fjernet det gamle view fra featuren og bpi_features.features.user_permission.inc, som slet ikke blev brugt.

Se http://platform.dandigbib.org/issues/1315.

### CODE REVIEW

File: bpi.module [hunk 1]
```diff
@@ -772,3 +772,138 @@ function bpi_workflow_by_machine_names($machine_names) {
...
+
+    // Add workflow.
+    $fields['workflow_node'] = array(
+      'id' => 'workflow_node',
+      'table' => 'views_entity_node',
+      'field' => 'workflow_node',
+      'label' => 'Operations',
+      'text' => 'change BPI state',
+    );
+    /*$display_options['sort'] = array(
```
Comment (cableman) [inline bpi.module]: Please remove dead/comment out code

File: bpi_features/bpi_features.features.user_permission.inc [hunk 2]
```diff
@@ -1,90 +0,0 @@
-<?php
```
Comment (cableman) [inline bpi_features/bpi_features.features.user_permission.inc:1]: I'm not sure that I understand why these permissions needs to be removed?

**Comment by cableman** [comment]
Please change the commit message to follow the standard format and link back to the issue.

Issue #1315 by bartvig: Message

File: bpi_features/bpi_features.features.user_permission.inc (hunk 2 above)
Comment (bartvig) [inline bpi_features/bpi_features.features.user_permission.inc:1]: This feature has never actually been used, i.e. it was never included in the feature module.
It was a mistake from the beginning, that this file was in the git repo.

Comment (holt83) [inline bpi_features/bpi_features.features.user_permission.inc:1]: Wouldn't they also just be overriden by the permissions setup by the Secure Permissions module?

**Comment by cableman** [comment]
Please re-base this PR so that it can merge. That may also remove the first 3 commits that do not relate to this issue.

**Comment by cableman** [comment]
Reviewed and approved

**Pull request merged**